.github/workflows/release-notes.yaml

```diff
@@ -7,24 +7,15 @@ on:
 jobs:
   preview:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.8]
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
     - run: |
-        git fetch --prune --unshallow
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+        git fetch --prune --unshallow --tags
```

Comment:
neither, I believe. could be wrong though, only started learning GitHub Actions.

Reply:
I don't know anything about the Actions, just seemed that if it's executing arbitrary commands then it could also fetch the repo itself, on which it's going to do some processing 🤷‍♂

Reply:
yes, this is the command that tells it to fetch the repo itself 😬